test/ui-test/src/drivers/wrappers/text_field.py

```diff
@@ -0,0 +1,25 @@
+import squish
+
+from .base_element import BaseElement
+
+
+class TextField(BaseElement):
+
+    @property
+    def text(self) -> str:
+        return str(self.object.text)
+
+    @text.setter
+    def text(self, value: str):
+        self.clear()
+        self.type_text(value)
+        assert squish.waitFor(lambda: self.text == value)
+
+    def type_text(self, value: str):
+        squish.type(self.object, value)
```

Comment:
the test fails to me with the following error on this method:

`Detail RuntimeError: Error in type() invocation: Can't replay type on object of type QQuickTextEdit_QML_195 because it is not ready: Quick item is clipped by a parent object  /Users/anastasiya/status-desktop/test/ui-test/src/drivers/wrappers/text_field.py:19`